.babelrc

```diff
@@ -9,12 +9,10 @@
           ],
           "extensions": [".ios.js", ".android.js", ".js", ".json"],
         "alias": {
-           "src": "./src",
-           "assets": "./src/assets",
-           "components": "./src/components",
-           "i18n": "./src/i18n",
-           "routes": "./src/routes",
-           "screens": "./src/screens"
+           "_assets": "./src/assets",
+           "_components": "./src/components",
```

Comment:
usually it is used with @ as for example  @components, is it reason why we use underscore?